pkg/agent/openflow/pipeline.go

```diff
@@ -433,340 +448,441 @@ func (c *client) DeleteOFEntries(ofEntries []binding.OFEntry) error {
 	return c.changeOFEntries(ofEntries, del)
 }
 
-// defaultFlows generates the default flows of all tables.
-func (c *client) defaultFlows() (flows []binding.Flow) {
-	for _, obj := range ofTableCache.List() {
-		table := obj.(binding.Table)
-		flowBuilder := table.BuildFlow(priorityMiss)
-		switch table.GetMissAction() {
-		case binding.TableMissActionNext:
-			flowBuilder = flowBuilder.Action().GotoTable(table.GetNext())
-		case binding.TableMissActionNormal:
-			flowBuilder = flowBuilder.Action().Normal()
-		case binding.TableMissActionDrop:
-			flowBuilder = flowBuilder.Action().Drop()
-		case binding.TableMissActionNone:
-			fallthrough
-		default:
-			continue
+func (c *client) defaultFlows(category cookie.Category) []binding.Flow {
+	cookieID := c.cookieAllocator.Request(category).Raw()
+	var flows []binding.Flow
+	for pipelineProto, pipeline := range c.pipelines {
+		// Generate default flow for every table for a pipeline.
+		for _, table := range pipeline.ListAllTables() {
+			flowBuilder := table.BuildFlow(priorityMiss).
+				Cookie(cookieID)
+
+			switch table.GetMissAction() {
+			case binding.TableMissActionNext:
+				flowBuilder = flowBuilder.Action().NextTable()
+			case binding.TableMissActionNormal:
+				flowBuilder = flowBuilder.Action().Normal()
+			case binding.TableMissActionDrop:
+				flowBuilder = flowBuilder.Action().Drop()
+			case binding.TableMissActionNone:
+				fallthrough
+			default:
+				continue
+			}
+			flows = append(flows, flowBuilder.Done())
+		}
+
+		// Generate flows in PipelineClassifierTable to forward traffic to different pipelines.
+		switch pipelineProto {
+		case pipelineIP:
+			for _, ipProtocol := range c.ipProtocols {
+				flows = append(flows, pipelineClassifyFlow(cookieID, ipProtocol, pipeline))
+			}
+		case pipelineARP:
+			flows = append(flows, pipelineClassifyFlow(cookieID, binding.ProtocolARP, pipeline))
```

Comment:
what's the default flow in table `PipelineClassifier`? I noticed below difference between old and new pipeline:
new:
```
 cookie=0x0, duration=3937.254s, table=PipelineClassifier, n_packets=0, n_bytes=0, priority=0 actions=NORMAL
```
old:
```
 cookie=0x1000000000000, duration=7245403.789s, table=0, n_packets=0, n_bytes=0, priority=0 actions=drop
```

Reply:
Added.